### DISCUSSION TITLE
Docs not responsive?

### DISCUSSION THREAD

**Discussion by seflm** [squidfunk/mkdocs-material#5598] [General]
Hello,

My docs seem to be unresponsive, the mobile view is almost identical to a large computer screen. The 🍔 icon is there for the menu, but the text size is really small. The docs are over at [qvex.eu/start](https://qvex.eu/start). Anyone any idea what do I do wrong?

I searched where I could and wasn't able to find an answer. Everywhere I read just that mobile (responsive) view is enabled by default.

I will post screenshots in comparison with other docs below.

Will appreciate any help/idea.

Thanks

![Screenshot_20230611_093328_com.android.chrome.png](https://github.com/squidfunk/mkdocs-material/assets/20502452/5ad4738a-bb0d-407b-a8b7-6504e4b99feb)

![Screenshot_20230611_093601_com.android.chrome.jpg](https://github.com/squidfunk/mkdocs-material/assets/20502452/95c42b18-87d3-4d1f-bad1-4376770e406c)

**Comment by squidfunk** [comment]
Following the link you shared, the site doesn't correctly define the [`viewport` meta property](https://github.com/squidfunk/mkdocs-material/blob/1c22ca42f25544f45e9dddac09266de0ee87f0a8/src/base.html#L32). I'm not sure what you did, but you probably overrode `site_meta`. Make sure to put everything in there that we defined, or things may not work as advertised 😉

**Reply by seflm** [reply]
Thank you! You are correct, I overrode the meta tag in an attempt of adding `noindex` and `nofollow`. Now I copied the entire meta block from the theme to a custom theme and just added what I needed. Thanks for your help :))
